lib/widgets/lightbox.dart

```diff
@@ -142,6 +142,28 @@ class _LightboxPageLayoutState extends State<_LightboxPageLayout> {
     });
   }
 
+  String _formatDate(DateTime messageDate) {
+    final now = DateTime.now();
+    final difference = now.difference(messageDate);
+    final locale = Localizations.localeOf(context).toString();
+
+    if (difference.inSeconds < 60) {
+      return "A few seconds ago";
+    } else if (difference.inMinutes < 60) {
+      return "${difference.inMinutes} ${difference.inMinutes == 1 ? 'minute' : 'minutes'} ago";
+    } else if (difference.inHours < 24) {
+      final time = DateFormat.jm(locale).format(messageDate);
+      return "Today at $time";
+    } else if (difference.inHours < 48) {
+      final time = DateFormat.jm(locale).format(messageDate);
+      return "Yesterday at $time";
+    } else {
+      final date = DateFormat('MMM d, yyyy', locale).format(messageDate);
+      final time = DateFormat('hh:mm a', locale).format(messageDate);
+      return "$date at $time";
+    }
```

Comment:
These are untranslated English strings. Instead, we should show strings in the user's chosen language.

Reply:
Thank you for the feedback, @chrisbobbe. 
Please can you provide a suggestion on how i should achieve that?

Reply:
I'm not able to prioritize that right now; this is a post-launch issue. Maybe there's a library out there that can handle this for us?

Reply:
Okay thank you, I will look that up.